scripts/common

```diff
@@ -9,17 +9,18 @@ bootstrap(){
 
 	export http_proxy=${APT_PROXY}
 
-	if [ "$(dpkg --print-architecture)" !=  "armhf" ] && [ "$(dpkg --print-architecture)" !=  "aarch64" ]; then
+	if [ "$(dpkg --print-architecture)" !=  "arm64" ]; then
 		BOOTSTRAP_CMD=qemu-debootstrap
 	fi
 
-	BOOTSTRAP_ARGS+=(--arch armhf)
+	BOOTSTRAP_ARGS+=(--arch arm64)
 	BOOTSTRAP_ARGS+=(--components "main,contrib,non-free")
-	BOOTSTRAP_ARGS+=(--keyring "${STAGE_DIR}/files/raspberrypi.gpg")
+	BOOTSTRAP_ARGS+=(--include gnupg)
+	BOOTSTRAP_ARGS+=(--keyring "${STAGE_DIR}/files/debian.gpg")
```

Comment:
I think because we are using debian repo packages rather than raspberrypi packages

Reply:
I understand that, was just intrigued why the upstream repo isn't doing the same.

Reply:
yep that is true

Reply:
The upstream repo doesn't use any keyring file, but I added a custom one because most distros based on debian (like Ubuntu) don't include this key by default, this is a workaround for this.